scripts/rollup/bundles.js

```diff
@@ -398,7 +398,7 @@ const bundles = [
     global: 'ReactDOMServer',
     minifyWithProdErrorCodes: false,
     wrapWithModuleBoundaries: false,
-    externals: ['react', 'react-dom'],
+    externals: ['react', 'util', 'crypto', 'async_hooks', 'react-dom'],
```

Comment:
Bun doesn't implement `createHook` in `async_hooks` (https://bun.com/docs/runtime/nodejs-apis) so I don't think this works. Need a dedicated implementation for methods imported from Fizz Server for Node.js.

Reply:
Oh yeah your right, Bun doesn't support it right now. I just copied the key for node externals as it was erroring in build, but I believe I can simply remove that async_hooks one and it should still work as the functions I'm exporting don't need it.